src/server/ServerListManager.cpp

```diff
@@ -113,6 +113,10 @@ void ServerListManager::initAll() NACOS_THROW(NacosException) {
         if (NacosStringOps::isNullStr(getNamespace())) {
             addressServerUrl = endpoint + ":" + NacosStringOps::valueOf(getEndpointPort()) + "/" +
                                getContextPath() + "/" + getClusterName();
```

Comment:
add a new params endpointContextPath to replace getContextPath() in addressserver url ,  if endpointContextPath is empty ,use getContextPath as compatible

Reply:
if contextPath and clusterName params start with "/" already , It is best to move out the starting “/” character